sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_integers.py

```diff
@@ -19,151 +19,147 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 import struct
+from typing import NoReturn, Tuple, Union
 
 
 class UInt64:
-    def __init__(self, value):
-        self.value = value & 0xFFFFFFFFFFFFFFFF
+    def __init__(self, value: int) -> None:
+        self._value: int = value & 0xFFFFFFFFFFFFFFFF
```

Comment:
I'm not sure if this is actually a behaviour change. Previous we were setting `self.value`, which was calling the setter code below, so it was actually doing: `self._value = (value & 0xFFFFFFFFFFFFFFFF) & 0xFFFFFFFFFFFFFFFF`
Let me know if this was originally intentional and I can revert.

Reply:
It should be fine, It was a redundant operation as value & 0xFFFFFFFFFFFFFFFF and (value & 0xFFFFFFFFFFFFFFFF) & 0xFFFFFFFFFFFFFFFF are the same thing. The point here is to represent the value as an unsigned 64 bit integer.